buildstockbatch/cloud/docker_base.py

```diff
@@ -235,7 +238,13 @@ def _prep_weather_files_for_batch(self, tmppath):
             )
             return epws_to_copy
 
-    def _prep_assets_for_batch(self, tmppath):
+    def _prep_assets_for_batch(self, tmppath, buildstock_csv_filename):
+        """Bundles together the assets used when running simulations, and stores them to tmppath/assets.tar.gz.
+
+        Note: This function should be called after sampling is run. If buildstock.csv is missing, it will fail.
```

Comment:
The `buildstock_csv_filename` param is here for this, so make this part of that param's doc; e.g.,
```suggestion
        :param buildstock_csv_filename: The buildstock.csv file, created by :func:`BuildStockSampler.run_sampling()`.
            It is only used to ensure that sampling occurs before preparing assets.
```